packages/conf/src/utils/replaceRootDir.ts

```diff
@@ -0,0 +1,21 @@
+export function replaceRootDir<T>(config: T, rootDir: string): T {
```

Comment:
As this function is used only for replacing th "<rootDir>" string in the config file, I think that `config` can be of type `LinguiConfig`

Reply:
Actually not, it not bind to shape of LinguiConfig. It can take anything, and return anything. And it's not directly compatible with LinguiConfig. This function called in a recursion, and subsequent calls could be with type "string" or array, etc. So if you put `config: LinguiConfig` typescript would complain

Reply:
try it yourself please.

Reply:
done